app/i18n/en/conf.php

```diff
@@ -37,6 +37,7 @@
 			'no_limit' => 'No limit',
 			'thin' => 'Thin',
 		),
+		'hide_nav_entries' => 'Hide the lower-left navigation buttons'
```

Comment:
There is a missing comma at the end of the line, plus all the entries in other languages. You could use the CLI tool to add them automatically.